test/perf/benchmarks/assets/compiler.test.ps1

```diff
@@ -2278,6 +2278,7 @@ function Start-PSPackage {
                     PackageVersion = $Version
                     PackageRuntime = $Runtime
                     PackageConfiguration = $Configuration
+                    IncludeSymbols = $IncludeSymbols
```

Comment:
Revert this. This is just a test file.

Reply:
But shouldn't we also test this?

Reply:
This is used only for testing the compiler. Please remove this.